service/worker/workerdeployment/version_activities.go

```diff
@@ -149,33 +149,32 @@ func (a *VersionActivities) CheckWorkerDeploymentUserDataPropagation(ctx context
 // CheckIfTaskQueuesHavePollers returns true if any of the given task queues has any pollers
 func (a *VersionActivities) CheckIfTaskQueuesHavePollers(ctx context.Context, args *deploymentspb.CheckTaskQueuesHaveNoPollersActivityArgs) (bool, error) {
```

Comment:
I'm realizing this args needs to be called `CheckTaskQueuesHavePollersActivityArgs` instead of `CheckTaskQueuesHaveNoPollersActivityArgs` cuz there's a big difference between the two

Reply:
I can fix that in my PR though